logging/syslogFormat.go

```diff
@@ -0,0 +1,367 @@
+package logging
+
```

Comment:
does this require a certain version of go?

Reply:
No Katie. Its just the package where the new file "syslogformat" added.